web_src/js/features/stopwatch.js

```diff
@@ -26,11 +26,34 @@ export function initStopwatch() {
     $(this).parent().trigger('submit');
   });
 
+  const poller = () => {
+    const fn = (timeout) => {
+      if (timeout <= 0) {
+        return;
+      }
+      setTimeout(() => {
+        const _promise = updateStopwatchWithCallback(fn, timeout);
+      }, timeout);
+    };
+
+    fn(notificationSettings.MinTimeout);
```

Comment:
```suggestion
    const timeout = notificationSettings.MinTimeout;
    if (timeout <= 0) {
      return;
    }
    setTimeout(() => {
      const _promise = updateStopwatchWithCallback(fn, timeout);
    }, timeout);
  };
```

Reply:
No this suggestion is wrong. You have dropped the definition of the fn that is being called.